### ISSUE TITLE
Document how to access private repos

### ISSUE THREAD

**Issue by Nyholm** [KnpLabs/php-github-api#477]
Question in #471 and the solution is in #440. 

This should be somewhere in the readme.

**Comment by josephmancuso** [comment]
yeah why isn't this in the docs? took me a long time to figure out that the code is:

```
$client = new \Github\Client(); 
$client->authenticate("oauth_api_token_here", null, \Github\Client::AUTH_HTTP_TOKEN);
$memberships = $client->currentUser()->repositories();
```

this took me a long time to figure out while looking through the docs. I think it should be in the doc/repos.md somewhere as that's the logical place to look for public and private repos retrieval. I believe the docs are broken up by api type so even just a note in those docs to look at the currentUser docs would be fine. Also the `currentuser` docs seem really incomplete.

i'll be able to put together a PR if needed

**Comment by acrobat** [comment]
Please open a PR with any improvements you've found necessary, this is the only way this project can improve! Thank!

**Comment by josephmancuso** [comment]
ok will do. thanks

**Comment by acrobat** [comment]
Fixed in #671
